packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx

```diff
@@ -19,7 +19,7 @@ export { VerticalButton } from './ToolbarVerticalButton.stories';
 export { FarGroup } from './ToolbarFarGroup.stories';
 
 export default {
-  title: 'Preview Components/Toolbar',
+  title: 'Toolbar',
```

Comment:
```suggestion
  title: 'Components/Toolbar',
```
![image](https://user-images.githubusercontent.com/9615899/212963690-45052503-b6f7-4374-90e7-8fd4a7f41b07.png)